### PR TITLE
[MRG] Signature standards and QES-validator

### PR DESCRIPTION
**Pull request by michelfernandez**
- Created new page signaturestandards for EN, DE, FR
- Changed URLs on config.yaml for EN, DE, FR
- Changed content on qes-validator for EN, DE, FR
- Signaturstandards will follow in next commit

### CODE REVIEW

File: site/config.yaml [hunk 1]
```diff
@@ -140,9 +142,12 @@ languages:
         - name: Help-Center
           url: https://help.skribble.com/english
           weight: 10
-        - name: QES explained
-          url: /qes-explained/
+        - name: Signaturstandards
```
Comment (betatim) [inline site/config.yaml]:
```suggestion
        - name: Signature standards
```
missing E in signature and split into two words

File: site/config.yaml [hunk 2]
```diff
@@ -140,9 +142,12 @@ languages:
...
+          url: /signaturestandards/
```
Comment (betatim) [inline site/config.yaml]:
```suggestion
          url: /signature-standards/
```
What do you think of using a hyphen between words like we do for `qes-explained` and the like? For me it is easier to read if we put a `-` where there would normally be a space (` `) between words.

File: site/content/english/signaturestandards.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: 100% legal certainty in court
 description: Skribble has the same legal certainty as signing by hand and offers 100% legal certainty in court.
 draft: false
-slug: qes-explained
+slug: signaturestandards
```
Comment (betatim) [inline site/content/english/signaturestandards.md]: If we decide to [change the slug in the navigation](https://github.com/BlockSigner/cove/pull/181/files#r328548963) we need to update this as well.

File: site/config.yaml (hunk 1 above)
Comment (michelfernandez) [reply site/config.yaml]: I corrected it. Thanks!

File: site/config.yaml (hunk 2 above)
Comment (michelfernandez) [reply site/config.yaml]: I leave it like this and Till looks at it again anyway. Then we can adjust it again.

File: site/content/english/signaturestandards.md (hunk 3 above)
Comment (michelfernandez) [reply site/content/english/signaturestandards.md]: I only committed one stand at noon and was not finished yet. Now all links should work. Thanks a lot for thinking!

**Comment by betatim** [comment]
This PR is now to #195's branch (magic-monday).

**Comment by fursinger** [comment]
Note: The conflicts were mainly caused by the renaming of the page **identity** to **enable-qes**.

**Pull request merged**